src/test/java/cz/cvut/kbss/study/service/security/SecurityUtilsTest.java

```diff
@@ -58,14 +59,17 @@ public class SecurityUtilsTest {
 
     private User user;
 
+    private RoleGroup roleGroupAdmin;
```

Comment:
as explained in other comment we are elevating user's permission to ADMIN and i do not understand why. I suggest to use something like `roleGroupUser` instead.